rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java

```diff
@@ -545,6 +546,7 @@ public void test_processPegoutsInBatch_after_RSKIP271() throws IOException {
         verify(provider, times(1)).setNextPegoutHeight(any(Long.class));
 
         verify(eventLogger, times(1)).logBatchPegoutCreated(any(), any());
+        verify(eventLogger, times(1)).logReleaseBtcRequested(any(), any(), any());
```

Comment:
Please verify here that the rskTxHash used in the log is the one from `rskTx` object.

Out of scope: Do we have another test verifying that post RSKIP146 the release_requested event includes the rsk transaction hash of the releaseRequest queue element?

Reply:
Your Recommendation Is Valid.. I Have Updated The Test

Reply:
We could also check that the amount logged is the total amount of the pegout requests.

It's probably a good idea to check the actual expected values when possible instead of using `any()`. It can help us catch possible errors

Reply:
Resolved